lib/analytics/course_statistics.rb

```diff
@@ -46,20 +46,19 @@ def articles_edited
 
   def find_contribution_ids
     revision_ids = []
-    page_ids = []
+    article_ids = []
     upload_ids = []
 
     @course_ids.each do |course_id|
       course = Course.find(course_id)
       revisions = course.revisions
       revision_ids << revisions.pluck(:id)
-      page_ids << revisions.pluck(:article_id)
+      article_ids << revisions.pluck(:article_id)
       upload_ids << course.uploads.pluck(:id)
     end
     @revision_ids = revision_ids.flatten.uniq
-    @page_ids = page_ids.flatten.uniq
+    @article_ids = article_ids.flatten.uniq
     @upload_ids = upload_ids.flatten.uniq
-    @article_ids = Article.where(namespace: 0, id: @page_ids).pluck(:id)
```

Comment:
Remove this changes the (under-tested) behavior of this method; it will include ids from outside mainspace.